Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/Confirm.hs

```diff
@@ -47,4 +47,4 @@ buildConfirmReq req = do
       { ..
       }
   where
-    castAddress Confirm.Address {..} = DBL.LocationAddress {areaCode = area_code, area = locality, ..}
+    castAddress Confirm.Address {..} = DL.LocationAddress {areaCode = area_code, area = locality, full_address = Nothing, ..}
```

Comment:
why full_address `Nothing`, and why not `fullAddress` in camelCase?

Reply:
Its an existing parameter which is present at BPP address and not in BAP address. hence while sending to BAP it should be nothing